pkg/installer/args.go

```diff
@@ -52,6 +52,7 @@ type Args struct {
 	csiProvisionerImage      string
 	nodeDriverRegistrarImage string
 	livenessProbeImage       string
+	Printer                  func(interface{})
```

Comment:
There is no need to output other than yaml format.

Reply:
installer will just print yaml